packages/align-deps/src/presets/microsoft/react-native/profile-0.73.ts

```diff
@@ -110,7 +114,7 @@ export const profile: Profile = {
   },
   netinfo: {
     name: "@react-native-community/netinfo",
-    version: "^11.0.1",
+    version: "^11.3.1",
```

Comment:
Strictly speaking, this should be 11.3.0. If we keep 11.3.1, we should remove it from the 0.74 profile:
```suggestion
    version: "^11.3.0",
```